pkg/plugins/commands.go

```diff
@@ -11,14 +13,50 @@ import (
 
 func Commands(l logger.Interface, s schema.Stage, fs vfs.FS, console Console) error {
 	var errs error
+	var wg sync.WaitGroup
+	wg.Add(len(s.Commands))
+	resultCh := make(chan error)
+
 	for _, cmd := range s.Commands {
-		out, err := console.Run(templateSysData(l, cmd))
-		if err != nil {
-			l.Error(out, ": ", err.Error())
-			errs = multierror.Append(errs, err)
+		switch strings.ToLower(s.RunType) {
+		case "ignore":
```

Comment:
looks like this should be:

```
errors: ignore
```
instead of run-type, ideally I expect something like:

```
background: true
errors: ignore|fatal
```

wdyt?

However, feel free to ignore, it looks good either ways to me

Reply:
Umm, that sounds nicer, let me check this further